### DISCUSSION TITLE
Is it possible to change the entire styling of the header with new css of our own ??

### DISCUSSION THREAD

**Discussion by venkateshpackianath** [thuliteio/doks#664] [Q&A]
I have been trying to change the header and footer that matches with our existing product. But I am not able to find a way to customize the header with whole change of styles. Is it possible ?

**Comment by h-enk** [comment]
Don't know what you're exactly trying to achieve, but you'd have to update:

-  `./assets/scss/layouts/_header.scss` (see `./layouts/partials/header/header.html` for structure and classes used)
- `./assets/scss/layouts/_footer.scss` (see `./layouts/partials/footer/footer.html` for structure and classes used)
